.werft/build.js

```diff
@@ -106,6 +106,7 @@ async function build(context, version) {
     const withInstaller = "with-installer" in buildConfig || masterBuild;
     const noPreview = "no-preview" in buildConfig || publishRelease;
     const registryFacadeHandover = "registry-facade-handover" in buildConfig;
+    const gcpStorage = "gcp-storage" in buildConfig;
```

Comment:
I'd much prefer something like `buildConfig.storage == "gcp"` (i.e. `/werft storage=gcp`). The boolean flags we introduced so far tend to get a bit out of hand. Also, should we ever add support for another object store, it's easier to add.